test/unit/lib/plugins/aws/package/lib/mergeIamTemplates.test.js

```diff
@@ -337,6 +340,12 @@ describe('lib/plugins/aws/package/lib/mergeIamTemplates.test.js', () => {
         expect(iamResource.Properties.RetentionInDays).to.be.equal(5);
         expect(iamResource.Properties.LogGroupName).to.be.equal(`/aws/lambda/${service}-dev-foo`);
       });
+
+      it('should support `provider.iam.role.tags`', () => {
```

Comment:
Let me know if you're okay with this - I know you were favouring single `runServerless` invocation per test.

Reply:
That's perfect :+1: